build/Targets/VSL.Imports.targets

```diff
@@ -1,6 +1,20 @@
 <Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 
+  <!-- Turn off implict binary and project reference copying to prevent projects
```

Comment:
Is this change independent of CPS?  Essentially something we should be doing in master?